.github/workflows/python-lint.yml

```diff
@@ -0,0 +1,27 @@
+name: python lint
+on: [push]
+jobs:
+  sdk-python-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: git clone
+        uses: actions/checkout@v3
+
+      - name: set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+
+      - name: lint with black
+        uses: psf/black@stable
```

Comment:
Could you use a specific tag here vs the stable branch directly? E.g. `[23.7.0](https://github.com/psf/black/releases/tag/23.7.0)`